plugins/woocommerce/src/Internal/TermCount.php

```
@@ -0,0 +1,143 @@
+	 * @since 11.1.0
+	 *
+	 * @internal
+	 *
+	 * @param int               $object_id  Object ID.
+	 * @param array<int|string> $terms      An array of object term IDs or slugs.
+	 * @param array<int|string> $tt_ids     An array of term taxonomy IDs.
+	 * @param string            $taxonomy   Taxonomy slug.
+	 * @param bool              $append     Whether to append new terms to the old terms.
+	 * @param array<int|string> $old_tt_ids The old array of term taxonomy IDs.
+	 */
+	public function handle_set_object_terms( $object_id, $terms, $tt_ids, $taxonomy, $append, $old_tt_ids ): void {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$terms'. (undefined)

(UnusedFormalParameter)